modules/statics/src/account.ts

```diff
@@ -982,3 +1014,63 @@ export function tavaxErc20(
     primaryKeyCurve
   );
 }
+
+/**
+ * Factory function for FIAT currency instances.
+ *
+ * @param name unique identifier of the currency
+ * @param fullName Complete human-readable name of the currency
+ * @param asset Asset which this coin represents. This is the same for both mainnet and testnet variants of a coin.
+ * @param decimalPlaces Number of decimal places this coin supports (divisibility exponent)
+ * @param network Network object for this coin
+ * @param prefix? Optional coin prefix. Defaults to empty string
+ * @param suffix? Optional coin suffix. Defaults to coin name.
+ * @param features? Features of this coin. Defaults to the DEFAULT_FEATURES defined in `AccountCoin`
+ * @param isToken? Whether or not this account coin is a token of another coin
+ * @param primaryKeyCurve The elliptic curve for this chain/token
+ */
+export function fiatToken(
```

Comment:
do we need `fiat()` / `tfiat()` factory functions? What's the difference between a fiat asset and a fiat token asset?

Reply:
Duplicated